documentation/docs/Get-started/intro.md

```diff
@@ -1,19 +1,22 @@
 ---
 sidebar_position: 1
-title: What is Monaco ?
+title: What is Monaco?
 ---
 
 Monaco is a CLI tool that automates the deployment of Dynatrace Monitoring Configuration to one or multiple Dynatrace environments.
 
 ## Why Monaco?
 
-Monaco’s self-service model enables development teams to set up monitoring and observability easily and efficiently, even for large scale applications. It eliminates the need for building custom monitoring solutions and reduces the manual work for monitoring teams.  
+Monaco’s self-service model enables development teams to set up monitoring and observability quickly and efficiently, even for large-scale applications. 
+It eliminates the need for building custom monitoring solutions and reduces the manual work for monitoring teams.  
 
-The Monitoring as Code (MAC) approach enables you to manage your Dynatrace environment monitoring tasks through configuration files instead of a graphical user interface. Configuration files allow you to create, update, and manage your monitoring configurations safely, consistently, and repetitively. They can be reused, versioned, and shared within your team.
+The Monitoring as Code (MAC) approach enables you to manage your Dynatrace environment monitoring tasks through configuration files instead of a graphical user interface. 
```

Comment:
```suggestion
The Monitoring as Code (Monaco) approach enables you to manage your Dynatrace environment monitoring tasks through configuration files instead of a graphical user interface. 
```

MAC is a pretty old acronym

Reply:
looks like this still says MAC at the moment? forgot a commit?

Reply:
No it does not, but I selected your other suggestion above. Have a look at the changed files directly.

Reply:
Different file :) 

As far as I can tell it still says MAC in `Get-started/intro.md`
https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/blob/145f16765d0e1bedf8804bb2df7c057fef862ad5/documentation/docs/Get-started/intro.md

You did remove it from `documentation.md`